src/Compilers/CSharp/Portable/Syntax/SyntaxFactory.cs

```diff
@@ -725,13 +725,13 @@ public static XmlEmptyElementSyntax XmlThreadSafetyElement()
         /// <summary>
         /// Creates a threadsafty element within an xml documentation comment.
         /// </summary>
-        /// <param name="static">Indicates whether static member of this class are safe for multi-threaded operations.</param>
-        /// <param name="instance">Indicates whether members of instances of this type are safe for multi-threaded operations.</param>
-        public static XmlEmptyElementSyntax XmlThreadSafetyElement(bool @static, bool instance)
+        /// <param name="isStatic">Indicates whether static member of this class are safe for multi-threaded operations.</param>
+        /// <param name="isInstance">Indicates whether members of instances of this type are safe for multi-threaded operations.</param>
```

Comment:
Consider changing wording of at least one of the  ```<param/>``` elements so the two descriptions are consistent.